dataland-qa-service/src/test/kotlin/org/datalandqaservice/services/QaServiceTest.kt

```diff
@@ -1,14 +1,18 @@
 package org.datalandqaservice.services
```

Comment:
The package of this file does not match the package of the tested source code. Also the name of the file should rather be DatalandQaServiceTest, to match the src file DatalandQaService. This did become obvious since actually another QaServiceTest.kt file exists in e2etests folder.

Reply:
On it

Reply:
Change is pushed